docs/HOL_Continuous-Deployment-Using-Custom-Agent/README.md

```diff
@@ -0,0 +1,121 @@
+        <img src="http://armviz.io/visualizebutton.png"/>
+    </a>
+
+    The VMs will be deployed to a Resource Group along with a virtual network (VNET) and some other required resources. You can 
+    delete the resource group in order to remove all the created resources at any time.
+
+4. You will need to select a subscription and region to deploy the Resource Group to and supply an admin username, password, and unique name for the machine. The machine will be a Standard A2.
+
+    ![](<media/set_arm_parameters.png>)
+
+    Make sure you make a note of the region as well as the username and password for the machine. Allow about 10 minutes for deployment and then another 10 minutes for the VSTS agent and MRP dependency configuration. 
+
+5. You will also need to specify the VSTS account to use (the DNS name before *visualstudio.com*) and a personal access token. If you don't have a personal access token, follow [this link](https://www.visualstudio.com/en-us/docs/setup-admin/team-services/use-personal-access-tokens-to-authenticate) to create one.
+
+6. When the deployment completes, you should see the following resources in the Azure Portal:
+
+    ![](<media/post_deployment_rg.png>)
+
+    Click on the "partsmrp" Public IP Address. Then make a note of the DNS name:
+
+    ![](<media/public_ip_dns.png>)
+
+    >**Note:** The lab requires several ports to be open, such as SSH ports and the Parts Unlimited MRP app port on the partsmrp machine. 
+	The ARM template opens these ports on the machine for you.
+
+### Task 2: Create release definition ###
+
+1. At the homepage of the PartsUnlimitedMRP team project in Visual Studio Team Services, click on the **Release** tab in the upper-left corner of the page. Then, click the **+** button on the left and choose **Create new release definition**.
+
+    ![](<media/create_release_definition.png>)
+
+2. In the **Create new release definition** dialog, choose an empty template then the OK button. 
+
+    ![](<media/create_empty_definition.png>)
+
+3. Keep the artifacts as **Build**, select the CI build definition that you used in the previous lab (such as "PartsUnlimited.CI"), check the checkbox to enable the **Continuous Deployment trigger**, and choose "MRP" as the  **agent queue**.
+
+    ![](<media/choose_source_queue_new_dialog.png>) 
+
+4. Click on the **Environment** keyword and rename the environment to be "Dev." Click on the pencil icon on the top of the definition and rename it to be PartsUnlimitedMRP.CD. 
+
+     ![](<media/change_environment_name.png>)
+
+5. Click on the **Add tasks** button and add a shell script task (under the Utility category). 
+
+	 ![](<media/add_shell_script.png>)
+
+6. Point to the **Deploy-MRP-App.sh** build artifact as the script path in the task. Then save the release definition. 
+
+	 ![](<media/add_script_path.png>)
+ 
```

Comment:
Missing one step before to click on save.
Set trigger on artifact source : 
![releasediftrig](https://cloud.githubusercontent.com/assets/8661347/17607277/c87a93b2-5ff2-11e6-974e-49e1e4ea8e30.png)